### PR TITLE
Add ACC_VALUE_TYPE class access modifier for Valhalla L-World

### PR DESCRIPTION
**Pull request by eric98zhang**
Signed-off-by: Eric Zhang <eric99.zhang@gmail.com>

### CODE REVIEW

**Comment by eric98zhang** [comment]
@tajila

**Comment by tajila** [comment]
@pshipton please take a look

File: runtime/oti/cfr.h [hunk 1]
```diff
@@ -827,6 +827,9 @@ typedef struct J9CfrClassFile {
 #define CFR_ACC_BRIDGE  					0x00000040
 #define CFR_ACC_VOLATILE  					0x00000040
 #define CFR_ACC_TRANSIENT  					0x00000080
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+#define CFR_ACC_VALUE_TYPE					0x00000100
```
Comment (pshipton) [inline runtime/oti/cfr.h]: Is it a problem this value conflicts with CFR_ACC_NATIVE?

Comment (tajila) [reply runtime/oti/cfr.h]: `CFR_ACC_NATIVE` is for method modifiers whereas `CFR_ACC_VALUE_TYPE` is for class modifiers

**Comment by pshipton** [comment]
Skip PR testing as it doesn't cover Valhalla changes.

**Pull request merged**